source/source/deengames/planetoid/entity/Player.hx

```diff
@@ -21,10 +23,26 @@ class Player
                             .add(new SpriteComponent('assets/images/butterfly.png'))
                             .add(new PositionComponent(0, 0))
                             .add(new KeyboardInputComponent(100))
-                            .add(new CameraComponent())
-                            .add(new MouseClickComponent());
-        var mouseClickComponent:MouseClickComponent = entity.get(MouseClickComponent);
-        mouseClickComponent.registerCallBack(function(x:Float,y:Float){trace(x,y);});
+                            .add(new CameraComponent());
+
+        var mouseClickComponent:MouseClickComponent = new MouseClickComponent();
+        mouseClickComponent.registerCallBack(function(x:Int,y:Int){trace("1",x,y);});
+        mouseClickComponent.registerCallBack(function(x:Int,y:Int){trace("2",x,y);});
+        var func = function(x:Int,y:Int){trace("3",x,y);};  
+        mouseClickComponent.registerCallBack(func);
+        mouseClickComponent.removeCallBack(func);
```

Comment:
No, it was just for testing that it works. I will move them to tests tomorrow in shaa' Allah